api/extensions/ext_celery.py

```diff
@@ -29,6 +29,9 @@ def __call__(self, *args: object, **kwargs: object) -> object:
     celery_app.conf.update(
         result_backend=app.config["CELERY_RESULT_BACKEND"],
         broker_connection_retry_on_startup=True,
+        broker_transport_options={
+            'global_keyprefix': app.config["CELERY_GLOBAL_PREFIX"]
```

Comment:
Carefully consider, suggest setting the `global_keyprefix` key here only when `CELERY_GLOBAL_PREFIX` is not empty.